### DISCUSSION TITLE
Can we get a downloadable link for the coverage report generated?

### DISCUSSION THREAD

**Discussion by padminib16** [coverlet-coverage/coverlet#1521] [General]
In my DevOps pipeline, I want to have a link where I want to be able to download the report in addition to viewing it too. Is that feature available?

**Comment by daveMueller** [comment]
Coverlet itself only calculates the coverage and saves the result in different output formats. You can chose the location where the result is stored. E.g. you can see this in the documentation of the [msbuild driver](https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/MSBuildIntegration.md#coverage-output).
In the pipeline you then usually expose the coverage result file as part of the artifacts like you would do it with e.g. test reports. The most build servers I worked with then let you easily download those artifacts.
For visualizing the reports, users often use the [ReportGenerator](https://github.com/danielpalme/ReportGenerator). But some build servers also support visualisation of coverage reports out of the box.
